helpers/requests/requests.go

```diff
@@ -0,0 +1,87 @@
+package requests
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// ConstructGetURL Constructs a URL to use in a GET request.
+// Queries are passed as a map of string arrays
+func ConstructGetURL(
+	rawURL string, queries map[string][]string, format string, values ...interface{}) (*url.URL, error) {
+
+	urlPath, err := url.Parse(rawURL)
+	if err != nil {
+		fmt.Printf("Unable to parse url %q", rawURL)
+		return nil, err
+	}
```

Comment:
Excessive logging (IMO). Better to only return the error and let the user of this function decide when to log in this case.

```suggestion
	if err != nil {
		return nil, err
	}
```